docs/mkdocs/documentation/deploy_kmod.md

```diff
@@ -42,11 +52,11 @@ The reconciliation loop for `Module` runs the following steps:
     1. go through `.spec.moduleLoader.container.kernelMappings` and find the appropriate container image name.
        If the kernel mapping has `build` or `sign` defined and the container image does not already exist, run the build
        and / or signing pod as required;
-    2. create a ModuleLoader `DaemonSet` with the container image determined at the previous step;
+    2. create a worker Pod pulling the container image determined at the previous step and running `modprobe`;
     3. if `.spec.devicePlugin` is defined, create a device plugin `DaemonSet` using the configuration specified under
        `.spec.devicePlugin.container`;
 4. garbage-collect:
-    1. existing `DaemonSets` targeting kernel versions that are not run by any node in the cluster;
+    1. obsolete device plugin `DaemonSets` that do not target any node;
     2. successful build pods;
```

Comment:
I remember some discussions about extending (or make it configurable) the time those build/sign pods are cleaned.

Do we already have that? If yes, it is worth mentioning it in the docs.

Reply:
We do not have that feature yet.